packages/generator-volto/package.json

```diff
@@ -108,7 +108,6 @@
     "fs-extra": "3.0.0",
     "gitly": "2.0.3",
     "mkdirp-then": "1.2.0",
-    "ms": "1.0.0",
```

Comment:
@sneridagh Also technically not backwards compatible, but it's very unlikely that someone is extending the generator so I wouldn't worry about it.